saleor/dashboard/graphql/utils.py

```diff
@@ -0,0 +1,13 @@
+import graphene
+
+
+def get_model_name(model):
+    model_name = model.__name__
+    return model_name[:1].lower() + model_name[1:]
+
+
+def get_model_type_and_fields(registry, model, return_field_name):
```

Comment:
I think the purpose of this function would be cleaner of it returned only mutation output fields, then we could name it `get_output_fields`.

Reply:
Also, these two functions here are very specific to the `saleor/dashboard/graphql/mutations.py` module, maybe they should live there for now? We won't use it anywhere else.